### ISSUE TITLE
How to set an expiration time?

### ISSUE THREAD

**Issue by godfatherzzx** [hootsuite/akka-persistence-redis#43]
for some reason, I need to set an expiration time for the snapshot. so how to configure the plugin?
Thanks for you work.

**Comment by ssong-van** [comment]
Do you mean the `timestamp` field in `SnapshotMetadata` when saving a snapshot? I think it's only used optionally when searching for a snapshot. It defaults to `0L` and this plugin should store the value as is.

**Comment by godfatherzzx** [comment]
No. For there is not enough redis space for the snapshot, so it is prudent to create a process to clean up some data after a period of time. Thanks for your reply.

**Comment by ssong-van** [comment]
Just so I understand correctly, are you suggesting to add ability in this plugin to specify a timeout (Redis TTL) on a snapshot? Snapshots are stored in a Sorted Set with `persistenceId` as key. Redis can only expire the entire Set, meaning all snapshots under the same `persistenceId` will be lost. An alternative solution could be to have an actor that cleans up snapshot by calling `deleteAsync` with a timestamp.
